docs/user-guide/batch.md

```diff
@@ -672,15 +689,17 @@ module load mpt
 cd $SLURM_SUBMIT_DIR
 
 # Set the number of threads to the CPUs per task
+# and propagate `--cpus-per-task` value to srun
 export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
+export SRUN_CPUS_PER_TASK=$SLURM_CPUS_PER_TASK
 
 # Launch the parallel job
 #   Using 36 threads per node
 #   srun picks up the distribution from the sbatch options
 srun --cpu-bind=cores ./my_openmp_executable.x
 ```
 
-This will run your executable "my_openmp_executable.x" in parallel on 36
+This will run your executable ``my_openmp_executable.x`` in parallel on 36
```

Comment:
I think a "code fence" uses only one backtick, two might break it?

Reply:
preview ok for me